plugins/org.jkiss.dbeaver.model.dashboard/src/org/jkiss/dbeaver/model/dashboard/registry/DashboardRegistry.java

```diff
@@ -82,6 +83,10 @@ private DashboardRegistry(IExtensionRegistry registry) {
         DBDashboardContext staticContext = new DBDashboardContext();
         for (DashboardProviderDescriptor dp : dashboardProviders.values()) {
             for (DashboardItemConfiguration dashboard : dp.getInstance().loadStaticDashboards(dp)) {
+                //fixme after resolving macOs issue
```

Comment:
Super-ugly hardcode.

If we want to disable some dashboard for certain OSes then we need to use OS mapping. 
See driver files mapping xml.